### DISCUSSION TITLE
nickname randomizer submod? (DONT CLOSE WITHOUT ANSWERING PLEASE!!!!)

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#8015] [general]
so, i saw a post on the mas Reddit page and it said smthn abt a "nickname randomizer submod". is that submod available for download or is it beta, or is it really old? (if it is pls still send me link)

**Comment by dreamscached** [comment]
That's not the right place to ask for a submod. This is the Monika After Story issues page and is not related to any submods.

**Comment by multimokia** [comment]
It's extremely old, I never *officially* released it, and it predates the existing player nicknames system for MAS, I wouldn't exactly recommend installing it at the moment as it doesn't account for player nicknames as set in MAS.

**Comment by Booplicate** [comment]
We don't just close issues, we move them to discussions where they belong to. You can continue there. Issues are for suggestions and bug reports.

**Comment by ghost** [comment]
well- people never answer discussions..-

**Comment by ghost** [comment]
they ignore  discussions

**Comment by Booplicate** [comment]
People answer when they have an answer and time to post it. After all it's a GitHub repository, not an online chat.

**Comment by ghost** [comment]
..sorry. am i annoying?

**Comment by Booplicate** [comment]
I just tried to clarify why the other issues were closed.

**Comment by ghost** [comment]
i know, i just feel annoying.
